staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -354,6 +354,15 @@ type KubeletConfiguration struct {
 	// Default: "None"
 	// +optional
 	CPUManagerPolicy string `json:"cpuManagerPolicy,omitempty"`
+	// singleProcessOOMKill, if true, will prevent the `memory.oom.group` flag from being set for container
+	// cgroups in cgroups v2. This causes processes in the container to be OOM killed individually instead of as
+	// a group. It means that if true, the behavior aligns with the behavior of cgroups v1.
+	// The default value is determined automatically when you don't specify.
+	// On non-linux such as windows, only nil is allowed.
+	// On cgroup v1 linux, the value set is ignored and always true.
```

Comment:
Please update the comments in `types.go` to reflect that "On cgroup v1 linux, only nil and true are allowed"

Validation code LGTM.

Reply:
As for `pkg/kubelet/apis/config/types.go`, I didn't find the point I have to fix.
```
	// SingleProcessOOMKill, if true, will prevent the `memory.oom.group` flag from being set for container
	// cgroups in cgroups v2. This causes processes in the container to be OOM killed individually instead of as
	// a group. It means that if true, the behavior aligns with the behavior of cgroups v1.
```